sheets.py

```diff
@@ -64,7 +102,24 @@ def _check_either_or_cols(self, either_or_cols, cols, sheet_name, message=""):
             raise MissingRequiredFieldError("quantity", sheet_name, message)
 
     def _parse_data(self, col_list, parsed_object, value, parsed_data, prop_params):
-        """Parse a data column and attach to it'a appropriate parsed object."""
+        """
+        Parse a data column and attach to it's appropriate parsed object.
+        Currently used in material sheet and process sheet
+
+        :param col_list: column list generated by spliting conditions (eg."molar_mass:data".split(':'))
+        :type col_list: list
+        :param parsed_object: a template dict object with keys like 'base','iden','prop','molar_mass','cond','process'
```

Comment:
I think the examples add confusion here. Try something like this instead.
```suggestion
        :param parsed_object: the dict object the data is being applied to
```